Update Mission statement (#236)
cncf/tag-contributor-strategy: mentoring/README.md

## Mission

> The Mentorship Working group

Reviewer: I had this wrong initially, but we should be consistent with the WG name
```suggestion
The Mentoring Working Group
```

Author: Updated!

- Encourages cloud native computing adoption by providing opportunities for a diverse group of new contributors to work on CNCF projects with experienced mentorship.
- Promotes growth and sustainability of projects through mentoring new and existing contributors.

> - Provides support and advice to the projects around mentorship initiatives.

Reviewer: ```suggestion
- Provides support and advice to CNCF projects around mentorship initiatives.
```

Author: Updated!

## Mission

> * Encourage cloud native computing adoption by providing opportunities for a diverse group of new contributors to work on CNCF projects with experienced mentorship.
> * Promote growth and sustainability for projects through mentoring new and existing contributors. 
> * Provide support and advice for the projects

Reviewer: I wonder if it's long enough we should add a short preamble before the points:
```suggestion
The Mentorship Working group
* Encourages cloud native computing adoption by providing opportunities for a diverse group of new contributors to work on CNCF projects with experienced mentorship.
* Promotes growth and sustainability of projects through mentoring new and existing contributors. 
* Provides support and advice to the projects around mentorship initiatives.
```

Reviewer 2: I agree, the conversation originally grew from 'community responsibility' so maybe something to reflect that? Or did you have something specific in mind?

Author: Updated as proposed.
Yes, we could also add a preamble around 'community responsibility'
